src/SGB_Command_Packet.md

```diff
@@ -2,7 +2,7 @@
 
 Command packets (aka Register Files) are transferred from the Game Boy to
 the SNES by using P14 and P15 output lines of the JOYPAD register
-(FF00h).  These same lines are also used to select the two rows in the
+(FF00).  These same lines are also used to select the two rows in the
```

Comment:
```suggestion
($FF00).  These same lines are also used to select the two rows in the
```

That said, replacing the hex number with a link to the joypad page would be better imo?

Reply:
The reason I didn't add a sigil here was because the style guide says it should be omitted in obvious cases, such as for addresses. I suppose this one isn't such an obvious address?

I agree that a link would be better though.

Reply:
The address thing is fair. As for the link, let's!